cms/djangoapps/contentstore/views/tests/test_bulksettings.py

```diff
@@ -211,19 +206,21 @@ def do_values_match(self, setting_types, child, computed_settings):
         """
 
         for setting_type in setting_types:
-            setting= getattr(child, setting_type)
+            setting = getattr(child, setting_type)
```

Comment:
Does this need a trailing comma, too?  Or is this tuple a case where it's syntactically incorrect to have a trailing comma?